### PR TITLE
added beta user group for recommendations

### PR DESCRIPTION
**Pull request by lsemesky**
@skhtet  - is this what you were thinking?

please do not merge this yet until I also have front-end logic in
@mannyrivera2010
@mleeBoeing

### CODE REVIEW

**Comment by lsemesky** [comment]
@mannyrivera2010 do I need to revert back to the setttings.py to get travis to build?

**Comment by lsemesky** [comment]
image
In order to test this, you will need to change this file according to the image (Setting the use auth server to True)

You will also need to checkout this repo: https://github.com/aml-development/demo-auth-service. you can do this by running the following
```
git clone git@github.com:ozone-development/demo-auth-service.git
pip install virtualenv
cd demo-auth-service/
virtualenv venv
source venv/bin/activate
pip install -r requirements.txt 
python manage.py runserver localhost:8003
```

File: plugins/default_authorization/main.py [hunk 1]
```diff
@@ -96,15 +96,18 @@ def _get_auth_data(self, username):
         user_data['is_org_steward'] = False
         user_data['is_apps_mall_steward'] = False
         user_data['is_metrics_user'] = False
+        user_data['is_beta_user'] = False
 
         for g in groups:
+            print(g + "\n")
```
Comment (mleeBoeing) [inline plugins/default_authorization/main.py]: Used for debug purposes?

Comment (lsemesky) [inline plugins/default_authorization/main.py]: good catch

File: ozpcenter/permissions.py [hunk 2]
```diff
@@ -62,6 +62,22 @@ def has_permission(self, request, view):
             return False
 
 
+class isBetaUser(permissions.BasePermission):
```
Comment (mannyrivera2010) [inline ozpcenter/permissions.py]: Is this needed anywhere?

Comment (lsemesky) [inline ozpcenter/permissions.py]: I removed the restriction on the similar endpoint upon your suggestion so I can remove this too if you think it best

**Comment by lsemesky** [comment]
Received verbal approval from @mannyrivera2010

**Pull request merged**